.github/workflows/storage_visualization.yaml

```
@@ -29,5 +31,12 @@ jobs:
           credentials_json: "${{ secrets.GCP_STORAGE_VISUALIZATION_CREDENTIALS }}"
           project_id: cpg-common
 
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+          cache: "pip" # caching pip dependencies
+
+      - run: pip install -r storage_visualization/requirements.txt analysis-runner
```

`submit.py` itself doesn't actually need the other dependencies. But might not hurt either way...

```suggestion
      - run: pip install analysis-runner
```